frontend/src/scenes/insights/InsightDisplayConfig.tsx

```diff
@@ -149,6 +154,26 @@ export function InsightDisplayConfig({ filters, activeView, disableTable }: Insi
                         <ChartFilter filters={filters} disabled={filters.insight === InsightType.LIFECYCLE} />
                     </span>
                 )}
+                {activeView === InsightType.TRENDS && (
+                    <span className="filter">
+                        <span className="head-title-item">y-axis format</span>
```

Comment:
I think this button should be to the left of Chart type, to minimize disruption to user workflows.
Also "y-axis" should be capitalized I think.

Reply:
Hmmm, it's only visible if chart type is line. if it's to the left then if you weren't on type "line" and changed to "line" it would appear to the left of the thing you're focussed on... 

I'm not sure which is better... watching it also makes me think maybe it should be disabled rather than hidden

## disabling the control

![2022-08-02 08 54 27](https://user-images.githubusercontent.com/984817/182322506-a78c3534-0bdd-4616-8467-06ef9d8f25ef.gif)

## hiding and showing to the left

![2022-08-02 08 53 24](https://user-images.githubusercontent.com/984817/182322509-9b9a669c-d595-4a0b-aed3-c72f98ff1064.gif)

## hiding and showing to the right

![2022-08-02 08 52 44](https://user-images.githubusercontent.com/984817/182322513-976d83eb-df5f-4c55-8bde-33357ec000ec.gif)

Reply:
I've left it appearing to the left and being disabled if not applicable

![2022-08-02 09 02 38](https://user-images.githubusercontent.com/984817/182324042-68f5082e-7727-4058-8c82-8f207a3688d9.gif)